stylesheets/main.css

```diff
@@ -0,0 +1,535 @@
+  padding: 0.4rem 1.4rem;
+  background: red;
+  color: #fff;
+  text-transform: uppercase;
+  font-weight: 600;
+  font-size: 1rem;
+  border: none;
+  border-radius: 2px;
+  width: 130px;
+  height: 50px;
+  display: inline-block;
+  cursor: pointer;
+}
+
+#videos #left-section #subscribe #subscribe-body {
+  margin-left: 63px;
+  flex-direction: column;
+}
+
+#videos #right-section #up-next a {
+  text-decoration: none;
+  color: #000;
+}
+
+#videos #right-section #related-videos a {
+  text-decoration: none;
+  color: #000;
+}
+
+#videos #left-section #subscribe #subscribe-body a {
+  margin-top: 0.8rem;
+  color: var(--primary-color);
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 0.9rem;
+}
+
+/* Main section of the page: Aside Section of the page */
+
+#videos #right-section {
+  flex-direction: column;
+  width: 25%;
+}
+
+#videos #right-section #up-next {
+  margin: 0.8rem 0;
+  padding: 0.8rem 0;
+  border-bottom: 1px solid #888;
+}
+
+#videos #right-section #up-next-heading {
+  font-size: 0.8rem;
+  color: var(--primary-color);
+}
+
+#videos #right-section #related-videos {
+  flex-direction: column;
+}
+
+#videos #right-section #related-videos .related-video-item {
+  margin-bottom: 6px;
+}
+
+#bottom-footer,
+#top-footer,
+#main-header,
+#header-left-section,
+#header-right-section,
+#videos,
+#videos #main-video,
+#videos #left-section #main-video #video-footer,
+#videos #left-section #main-video #video-footer #videos-footer-lower,
+#videos #left-section #main-video #video-footer #videos-footer-lower #social-icons,
+#videos #left-section .comments,
+#videos #left-section .comments #comment-heading,
+#videos #left-section .comments #comment-heading #top-comment-heading,
+#videos #left-section .comments #comment-heading #bottom-comment-heading,
+#videos #left-section .comments .comment,
+#videos #left-section #subscribe,
+#videos #left-section #subscribe #subscribe-heading,
+#videos #left-section #subscribe #subscribe-heading #left-subscribe-heading,
+#videos #left-section #subscribe #subscribe-body,
+#videos #right-section,
+#videos #right-section #up-next,
+#videos #right-section #up-next-heading,
+#videos #right-section #up-next-heading #autoplay,
+#videos #right-section #related-videos,
+#videos #right-section #related-videos .related-video-item {
+  display: flex;
+}
+
+#main-header,
+#header-right-section,
+#videos,
+#videos #left-section #main-video #video-footer #videos-footer-lower,
+#videos #left-section #main-video #video-footer #videos-footer-lower #social-icons,
+#videos #left-section #subscribe #subscribe-heading,
+#videos #left-section #subscribe #subscribe-heading #left-subscribe-heading,
+#videos #right-section #up-next-heading,
+#videos #right-section #up-next-heading #autoplay {
+  justify-content: space-between;
+}
+
+#videos #right-section #up-next #up-next-video,
+#videos #right-section #related-videos .related-video .related-video-image {
+  position: relative;
```

Comment:
[REQUIRED !!] You are using a lot of unnecessary selectors to implement styling to an element. You can remove most of the parent selectors as you are using ids mostly. Generally, we can use 2 or 3 selectors to apply styling to an element. Please read [this](https://speckyboy.com/good-bad-css-practice) article, it will be helpful. [pathright requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png).
TIPS: Use class selector instead of  an ID selector if you want to implement same styling to multiple elements.

Reply:
Hello, thanks for the feedback. Please note that all changes suggested have been made.